config.md

```diff
@@ -69,7 +69,7 @@ The runtime MUST mount entries in the listed order.
 ## Process configuration
 
 * **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process. Defaults to false.
-* **`cwd`** (string, required) is the working directory that will be set for the executable.
+* **`cwd`** (string, required) is the working directory that will be set for the executable. This value MUST be an absolute path.
```

Comment:
On Fri, Jan 15, 2016 at 10:26:51AM -0800, Doug Davis wrote:
> > -* **`cwd`** (string, required) is the working directory that will be set for the executable.
> > +* **`cwd`** (string, required) is the working directory that will be set for the executable. This value MUST be an absolute path.
> 
> Whether cwd is required or not is a different issue/PR so its out of scope of this PR.
> 
> Given its required, I think abs path is the only valid choice…

I agree on both points.  But since #307 is only two days old (from
proposal to merge), I thought I'd give folks another chance to revisit
before too much time was sunk in implementation details (like this PR)
for a feature I don't think we need (#307).  If #307 is already locked
in, then yeah, I think this PR is the only sensible way forward.